core/src/main/java/fr/sncf/osrd/infra/implementation/tracks/undirected/TrackSectionImpl.java

```diff
@@ -20,6 +20,8 @@ public class TrackSectionImpl implements TrackSection {
     private final LineString geo;
     private final LineString sch;
     private final ImmutableRangeMap<Double, LoadingGaugeConstraint> loadingGaugeConstraints;
+    private int trackNumber = 0;
+    private int lineCode = 0;
```

Comment:
These can be final

Reply:
It cannot since it's used in the constructor

Reply:
try removing the default value of zero